sickrage/helper/common.py

```diff
@@ -22,6 +22,7 @@
 import re
 import sickbeard
 from fnmatch import fnmatch
+from sickbeard import logger
```

Comment:
You can't import logger in common.py.  It creates a circular dependancy loop.  The only option for logging in common.py is to use python standard logging and add it to the logger like we do for external libraries like subliminal.